### PR TITLE
Pass all 'other' event types to single function in makeRunnerCallback

### PR DESCRIPTION
**Pull request by alexmojaki**
As suggested in https://github.com/dodona-edu/papyros/pull/112#discussion_r844075522

### CODE REVIEW

File: test/worker.ts [hunk 1]
```diff
@@ -23,13 +23,15 @@ Comlink.expose({
...
         output: outputCallback,
+        other: (type, data) => type + "-" + JSON.stringify(data),
       });
       const pyodide = await pyodidePromise;
       if (extras.interruptBuffer) {
         pyodide.setInterruptBuffer(extras.interruptBuffer);
       }
       const runner = pyodide.pyimport("python_runner").PyodideRunner();
       runner.set_callback(callback);
+      pyodide.pyimport("builtins").runner = runner;
```
Comment (winniederidder) [inline test/worker.ts:34]: This is to make running tests more compact, I presume?

**Review by winniederidder — APPROVED** [review]
This change will make it more flexible if users want a catch-all. However, maybe you want to offer a hybrid approach, i.e. if a callback for the type exists, use that callback, otherwise use the catch-all?

**Comment by alexmojaki** [comment]
> However, maybe you want to offer a hybrid approach, i.e. if a callback for the type exists, use that callback, otherwise use the catch-all?

Sounds overcomplicated for users.

File: test/worker.ts (hunk 1 above)
Comment (alexmojaki) [reply test/worker.ts:34]: It was the easiest way to make `runner` available in Python code to test `runner.callback(...)` without having to write a Python file which does things like subclass PyodideRunner.

**Pull request merged**